src/utils/bets.ts

```diff
@@ -17,6 +17,7 @@ const getBetStatus = (
 ): BetStatus => {
   const hasResult =
     signedBetData.winningPropositionId || signedBetData.marketResultAdded;
+  if (+bet.payoutAt > Math.floor(Date.now() / 1000)) return "PENDING";
   if (!hasResult && !bet.settled) return "PENDING";
```

Comment:
Better to use an enum

Reply:
these are string literals as part of the `BetStatus` type which @pondpiu wrote - i believe he had a reason for doing so